source/style-guide/markup/format/roles.txt

```diff
@@ -0,0 +1,16 @@
+=====
+Roles
+=====
+
+Roles are a type of directive that are used inline and can be used to create
```

Comment:
[suggestion for minimalism]

Could we say:

"Roles are inline directives used to create cross-references or apply formatting."

Reply:
I'm fine with the change, but i wonder whether "inline directive" might cause ambiguity as to whether it's a term or just a directive that has inline placement.

Reply:
That's a fair point. "inline directives" might be a bit too convoluted / jargony. I'm happy to leave this as-is as even though it is a bit wordier, it may be easier to understand than my suggestion.